core/data/src/main/java/com/google/samples/apps/nowinandroid/core/data/util/ConnectivityManagerNetworkMonitor.kt

```diff
@@ -86,7 +82,8 @@ class ConnectivityManagerNetworkMonitor @Inject constructor(
         VERSION.SDK_INT >= VERSION_CODES.M ->
             activeNetwork
                 ?.let(::getNetworkCapabilities)
-                ?.hasCapability(NetworkCapabilities.NET_CAPABILITY_INTERNET)
+                ?.hasCapability(NET_CAPABILITY_INTERNET)
```

Comment:
Same a prior comment, please don't us a static import for `NetworkCapabilities.NET_CAPABILITY_INTERNET`.